rep-0149.rst

```diff
@@ -0,0 +1,1091 @@
+REP: 149
+Title: Package Manifest Format Three Specification
+Author: Dirk Thomas
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 11-Oct-2017
+
+Outline
+=======
+
+#. Abstract_
+#. Motivation_
+#. Rationale_
+#. `Data Representation`_
+#. Compatibility_
+#. Schema_
+#. References_
+#. Copyright_
+
+.. raw:: html
+
+  The current draft highlights significant changes compared to REP-0140 [2]_
+  in <font color="red">red</font>.
+  Trivial changes appear in <font color="green">green</font>.
```

Comment:
Not a content thing, but could I ask for the deuteranomalous and deuteranopes among us to use a different colour here?

Reply:
Thanks for raising this. I should have caught it but since GitHub filters out html tags the draft I was reviewing wasn't colorized.
My CSS overrides for GitHub.com use `#b85c00` in place of dark red and `#3269a0` for dark green. It's also worth noting that these indicators will be of no utility for readers using text-to-speech. So perhaps additional indicators should be used for significant changes.

Reply:
I switched the green to blue: 2e7172fb5e658afa1d9e5e6a06fee73948ea962a

For the final document the coloring will likely be removed.